src/Strategy/Conda/CondaList.hs

```diff
@@ -28,11 +29,14 @@ condaListCmd =
 buildGraph :: [CondaListDep] -> Graphing Dependency
 buildGraph deps = Graphing.fromList (map toDependency deps)
   where
+    makeV2Dep :: Text -> Text -> Text -> Text
```

Comment:
Where does the name `V2Dep` come from? I don't love this, especially since we plan to support the other dependency format indefinitely. I think something like `depWithlocation`would be more clear.

Reply:
I called it that because I felt it would make it easy to disambiguate fields if we were to add another format in the future. I can see how this would make it seem like we planned to deprecate the old one. I will rethink this, `depWithLocation` is probably more descriptive.